llvm/lib/Object/COFFObjectFile.cpp

```diff
@@ -1134,6 +1137,10 @@ StringRef COFFObjectFile::getFileFormatName() const {
     return "COFF-ARM64X";
   case COFF::IMAGE_FILE_MACHINE_R4000:
     return "COFF-MIPS";
+  case COFF::IMAGE_FILE_MACHINE_RISCV32:
+    return "COFF-RISCV32";
```

Comment:
Is there some standard these need to conform to (if so, where is this documented?) or are they whatever we want them to be?